requirements.in

```diff
@@ -2,6 +2,8 @@
 # See LICENSE file for licensing details.
 
 jinja2
-ops
+ops<2
```

Comment:
We're there, just by standard we have to always add a note when we pin versions, just mention here that v2.31 of dex uses this ops version and we are good. After that I can approve the PR, LGTM.

Reply:
Done.